index.ts

```diff
@@ -389,7 +390,8 @@ collect.set('isBranches', [
 export const isUserProfile = (): boolean => exists('.user-profile-nav');
 
 export const isUserProfileMainTab = (): boolean =>
-	isUserProfile() && !new URLSearchParams(location.search).get('tab');
+	isUserProfile() &&
+	(!new URLSearchParams(location.search).get('tab') || new URLSearchParams(location.search).get('tab') === 'overview');
```

Comment:
How do you get to https://github.com/fregante?tab=overview?

Reply:
https://github.com/fregante?tab=wawawewa also points to the main tab.
 
Since `isUserProfile` is dom-based anyway, let's use a single `exists` in this check, without `isUserProfile`

Reply:
```js
exists('[aria-label="User profile"] > .selected:first-child')
```

Reply:
> How do you get to [@fregante?tab=overview](https://github.com/fregante?tab=overview)?

Click load activity